shared/src/main/kotlin/com/egm/stellio/shared/util/JsonLdUtils.kt

```diff
@@ -48,8 +48,9 @@ object JsonLdUtils {
     const val JSONLD_ID = "@id"
     const val JSONLD_TYPE = "@type"
     const val JSONLD_VALUE_KW = "@value"
-    val JSONLD_ENTITY_MANDATORY_FIELDS = setOf("id", "type", "@context")
-    val JSONLD_ENTITY_CORE_PROPERTIES = setOf("id", "type", "@context")
+    const val JSONLD_CONTEXT = "@context"
+    val JSONLD_EXPANDED_MANDATORY_FIELDS = setOf(JSONLD_ID, JSONLD_TYPE, JSONLD_CONTEXT)
+    val JSONLD_COMPACTED_MANDATORY_FIELDS = setOf("id", "type", JSONLD_CONTEXT)
```

Comment:
I would leave the `Entity` word in the variable name as there are mandatory fields for other NGSI-LD data types

Reply:
ok